subsys/bluetooth/services/ras/rreq/Kconfig.ras_rreq

```diff
@@ -23,4 +23,8 @@ config BT_RAS_RREQ_MAX_ACTIVE_CONN
 	help
 	  The number of simultaneous connections with an instance of RAS RREQ
 
+# GATT DM module requires heap
+config HEAP_MEM_POOL_SIZE
+	default 512
```

Comment:
no, do not do this, add a new symbol with the prefix `HEAP_MEM_POOL_ADD_SIZE_` and set it